### PR TITLE
Add download instructions for pretrained model in dynamic quantization tutorial

### LINKED ISSUE #3254: How to download pretrained word language quantized model?
In the word language quantized model tutorial, we assume we already have pretrained model.
But where can we download the model?

https://github.com/pytorch/tutorials/blob/main/advanced_source/dynamic_quantization_tutorial.py#L151-L157

### PR DESCRIPTION
**Pull request by Ashish-Soni08**
Fixes #3254
- Added wget command to download word_language_model_quantize.pth
- Included note about file placement  
- Improved tutorial usability by providing missing download information
- Referenced official Makefile download method

## Description
This PR addresses issue #3254 by adding clear download instructions for the pretrained model file `word_language_model_quantize.pth` required by the dynamic quantization tutorial. The tutorial previously assumed users already had the model file without explaining where to download it.

The fix adds a clear note with a wget command using the same trusted URL found in the repository's Makefile (lines 65-66: https://github.com/pytorch/tutorials/blob/main/Makefile#L65), along with guidance on file placement. This ensures users get the same reliable download method that the official build system uses.

## Checklist
- [x] The issue that is being fixed is referred in the description (see above "Fixes #3254")
- [x] Only one issue is addressed in this pull request
- [x] Labels from the issue that this PR is fixing are added to this pull request
- [x] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: advanced_source/dynamic_quantization_tutorial.py [hunk 1]
```diff
@@ -129,15 +129,27 @@ def tokenize(self, path):
...
-# by training for five epochs using the default settings in the word language model
-# example.
+# that is applied after a model has been trained. Therefore, we'll simply
+# load some pretrained weights into this model architecture; these
+# weights were obtained by training for five epochs using the default
+# settings in the word language model example.
+#
+# **Note:** Before running this tutorial, download the required pretrained model:
+#
+# .. code::
```
Comment (svekars) [inline advanced_source/dynamic_quantization_tutorial.py]:
```suggestion
# .. code-block:: bash
```

**Review by svekars — APPROVED** [review]

File: advanced_source/dynamic_quantization_tutorial.py [hunk 2]
```diff
@@ -129,15 +129,27 @@ def tokenize(self, path):
 
 corpus = Corpus(model_data_filepath + 'wikitext-2')
 
+######################################################################
+# 3. Load the pretrained model
+# -----------------------------
+#
```
Comment (svekars) [inline advanced_source/dynamic_quantization_tutorial.py]: You have this below already
```suggestion
```

Comment (svekars) [reply advanced_source/dynamic_quantization_tutorial.py]: There is also an issue with formatting: https://docs-preview.pytorch.org/pytorch/tutorials/3379/advanced/dynamic_quantization_tutorial.html#load-in-the-text-data

Comment (Ashish-Soni08) [reply advanced_source/dynamic_quantization_tutorial.py]: Hey @svekars How should i go about making theses modifications?

File: advanced_source/dynamic_quantization_tutorial.py [hunk 3]
```diff
@@ -134,10 +134,18 @@ def tokenize(self, path):
...
-# that is applied after a model has been trained. Therefore, we'll simply load some
-# pretrained weights into this model architecture; these weights were obtained
-# by training for five epochs using the default settings in the word language model
-# example.
+# that is applied after a model has been trained. Therefore, we'll simply
+# load some pretrained weights into this model architecture; these
+# weights were obtained by training for five epochs using the default
+# settings in the word language model example.
+#
+# **Note:** Before running this tutorial, download the required pretrained model:
```
Comment (svekars) [inline advanced_source/dynamic_quantization_tutorial.py]:
```suggestion
# Before running this tutorial, download the required pre-trained model:
```
